### PR TITLE
rustc-perf triage for 2024-04-09

### PR DESCRIPTION
**Pull request by pnkfelix**

### CODE REVIEW

File: draft/2024-04-10-this-week-in-rust.md [hunk 1]
```diff
@@ -99,7 +99,19 @@ If you are an event organizer hoping to expand the reach of your event, please s
 
 ### Rust Compiler Performance Triage
 
-<!-- Perf results go here -->
+A quiet week; all the outright regressions were already triaged (the one biggish one was #122077, which is justi\
+fied as an important bug fix). There was a very nice set of improvements from PR #122070, which cleverly avoids \
+a lot of unnecessary allocator calls when building an incremental dep graph by reusing the old edges from the pr\
+evious graph.
```
Comment (mariannegoldin) [inline draft/2024-04-10-this-week-in-rust.md:105]: remove line breaks:
```suggestion
A quiet week; all the outright regressions were already triaged (the one biggish one was #122077, which is justified as an important bug fix). There was a very nice set of improvements from PR #122070, which cleverly avoids a lot of unnecessary allocator calls when building an incremental dep graph by reusing the old edges from the previous graph.
```

File: draft/2024-04-10-this-week-in-rust.md [hunk 2]
```diff
@@ -99,7 +99,16 @@ If you are an event organizer hoping to expand the reach of your event, please s
 
 ### Rust Compiler Performance Triage
 
-<!-- Perf results go here -->
+A quiet week; all the outright regressions were already triaged (the one biggish one was #122077, which is justified as an important bug fix). There was a very nice set of improvements from PR #122070, which cleverly avoids a lot of unnecessary allocator calls when building an incremental dep graph by reusing the old edges from the previous graph.
+
+Triage done by **@pnkfelix**.
+Revision range: [3d5528c2..86b603cd](https://perf.rust-lang.org/?start=3d5528c287860b918e178a34f04ff903325571b3&\
+end=86b603cd792b3f6172ba4f676d7b586c1af7630a&absolute=false&stat=instructions%3Au)
```
Comment (mariannegoldin) [inline draft/2024-04-10-this-week-in-rust.md:106]: remove line break in url:
```suggestion
Revision range: [3d5528c2..86b603cd](https://perf.rust-lang.org/?start=3d5528c287860b918e178a34f04ff903325571b3&end=86b603cd792b3f6172ba4f676d7b586c1af7630a&absolute=false&stat=instructions%3Au)
```

**Review by mariannegoldin — APPROVED** [review]
Thank you! Approved with a formatting change to remove line breaks.

**Comment by pnkfelix** [comment]
thanks for the formatting fixes. I must have cut-and-pasted stuff out of my terminal instead of my web browser without thinking about it.

**Pull request merged**